styles.css

```diff
@@ -0,0 +1,39 @@
+.calculator {
+  height: 230 px;
+  width: 202px;
+  background-color: rgb(49, 48, 48);
```

Comment:
It is not common to use rgb if there are no transparency (the rgba). I would go with hex.

Reply:
done